### ISSUE TITLE
Increase add_bid/delegate costs

### ISSUE THREAD

**Issue by mpapierski** [casper-network/casper-node#1911]
Ref #1902 

After merging #1910 we should increase add_bid/delegate entrypoints costs to be greater than `create_purse` host function call. Without this when 1.4.0 will be out, then add_bid would cost the same as create_purse call, but conceptually auction's entry points are more complex than create_purse itself

**Comment by cspramit** [comment]
@mpapierski is this suppose to be in 1.4?

**Comment by AlexanderLimonov** [comment]
Keeping in mind that we set the relevant costs to 2.5 _specifically_ because it's the account minimum, thereby _making it easier to remember_, one could proceed similarly here and increase it to a multiple, say 5. However, this may be seen as a big price hike, so perhaps 3 (still easy to remember, I hope) would do.

Fundamentally, these costs will have to be recalculated once I finally get to it (we still have placeholder costs for host side functions that were added after the 1.0 cost table was created, then we rescaled the costs, etc.), although we may want to maintain some form of "special" costs for frequently used functions. This also interacts with the proposal to introduce baseline costs for all deploys. The bottom line, keep it simple for now because it will have to change later.

@mpapierski
